### DISCUSSION TITLE
Is there a way I can access pocketbase types in my project.

### DISCUSSION THREAD

**Discussion by shanthisa** [pocketbase/pocketbase#715] [Q&A]
I can see that the types are exported to the `pocketbase.es.d.mts` file. What is the best way to import it into my project? I tried using the `/// <reference types="pocketbase/dist/pocketbase.es"/>` line on a separate `pocketbase.d.ts` file hoping that the typescript compiler would pick it up but it did not work. I would like to use the types returned by the SDK in my project. Especially the auth interface types.

**Comment by ganigeorgiev** [comment]
The available `types` property in the package.json - https://github.com/pocketbase/js-sdk/blob/master/package.json#L18, should be enough to instruct the TS compiler from where to load the package types.

Make sure that you are using the EcmaScript bundle, otherwise depending on your ts config it may fallback to the legacy commonjs one.

_You can find similar issues in https://github.com/pocketbase/js-sdk/issues/5 and https://github.com/pocketbase/js-sdk/issues/34._

**Reply by benallfree** [reply]
@ganigeorgiev Speaking of this, I did run into a couple cases where the PB type I needed was not exported. Ok to send PRs for that kind of thing?
